### PR TITLE
Fix some ESlint warnings

### PR DESCRIPTION
**Pull request by koba04**
CONTRIBUTING.md should do `grunt lint` to your codes.
But, It has many warnings (max-len) now.

```
➜ grunt lint
Running "eslint" task
>> Lint passed (but may contain warnings)
:
✖ 104 problems (0 errors, 104 warnings)
```

This pull request is fixing it.

```
 ➜  grunt lint
Running "eslint" task
>> Lint passed (but may contain warnings)

Done, without errors.
```

All warnings are about 'max-len'.
Therefore, this changes are for formatting codes.

### CODE REVIEW

**Comment by zpao** [comment]
Ok, I sort of wish you had gotten in touch first. There are things I don't care about and strictly listening to lint often leads to uglier code. This is why it's only a warning, not an error.

Many of these are fine but I'm going to ask you to revert some of the changes.

File: src/browser/ui/ReactDOMComponent.js [hunk 1]
```diff
@@ -275,6 +280,7 @@ ReactDOMComponent.Mixin = {
    * @param {object} context
    * @return {string} Content markup.
    */
+   /* eslint-enable max-len */
```
Comment (zpao) [inline src/browser/ui/ReactDOMComponent.js]: I actually don't want any of these in here. It's pure noise and we'll forget to remove them if we change types. I do wish there was an option to eslint to ignore the long lines in comments - I would rather fork the rule and make that happen than have these in there.

File: src/browser/ui/React.js [hunk 2]
```diff
@@ -116,7 +116,8 @@ if (__DEV__) {
 
     // If we're in IE8, check to see if we are in combatibility mode and provide
     // information on preventing compatibility mode
-    var ieCompatibilityMode = document.documentMode && document.documentMode < 8;
+    var ieCompatibilityMode = document.documentMode &&
+                              document.documentMode < 8;
```
Comment (zpao) [inline src/browser/ui/React.js]: Put the whole RHS on a single new line, that's prefereable to this (there's something similar elsewhere but it's not in `src/` so I don't really care)

File: src/browser/ui/dom/DOMProperty.js [hunk 3]
```diff
@@ -106,19 +106,40 @@ var DOMPropertyInjection = {
...
+        checkMask(
+          propConfig,
+          DOMPropertyInjection.HAS_POSITIVE_NUMERIC_VALUE
+        );
       DOMProperty.hasOverloadedBooleanValue[propName] =
-        checkMask(propConfig, DOMPropertyInjection.HAS_OVERLOADED_BOOLEAN_VALUE);
+        checkMask(
+          propConfig,
+          DOMPropertyInjection.HAS_OVERLOADED_BOOLEAN_VALUE
+        );
```
Comment (zpao) [inline src/browser/ui/dom/DOMProperty.js]: Please revert all of these. This is strictly worse IMO.

**Comment by koba04** [comment]
@zpao 
Thank you for your feedback.
I've got it priority to readability than strictly lint.

I would be glad if you review the code.
This changes are for reducing some warnings. (not all...)

```
➜ grunt lint
Running "eslint" task
>> Lint passed (but may contain warnings)
:
✖ 47 problems (0 errors, 47 warnings)
```

About remaining warnings, I'd like to propose as a issue if I have some ideas.
Thank you.

**Comment by zpao** [comment]
We're all caught up on linting, these probably got fixed by @bgw or somebody. Thanks though!

**Comment by koba04** [comment]
Thanks!

**Comment by bgw** [comment]
@zpao I don't think they got fixed. I think we just disabled the `max-len` rule.

**Comment by zpao** [comment]
Either way, lint doesn't complain :)

**Pull request closed without merging**